### ISSUE TITLE
Import/export SDCLOGGING.DAT

### ISSUE THREAD

**Issue by FLO-2DJJ** [FLO-2DSoftware/qgis-flo-2d-plugin#1267]
I think is enough to import/export the data to/from the user Storm Drain Nodes layer. No need to schematize it.

- [x] An issue is when to do the import: at **Import data (*.DAT) files**?  at **Import SWMM.INP** in the SD Editor Widget? _Answer: In both_.
- [x] Ask the user before importing? _Answer: No_
- [x] Replace the old one when exporting if all values are zero? _Answer: Yes_

**Comment by FLO-2DJJ** [comment]
Task completed:
- [ ] Import data (.DAT) files command: reads SDCLOGGING.DAT and updates Storm Drain Nodes tables/layer's fields swmm_clogging_factor and swmm_time_for_clogging.
- [ ] Import SWMM.INP... (Create new Storm Drains): reads SDCLOGGING.DAT and updates Storm Drain Nodes tables/layer's fields swmm_clogging_factor and swmm_time_for_clogging.
- [ ] Export data (.DAT) files command: writes SDCLOGGING.DAT if there are inlets/junctions with fields swmm_clogging_factor > 0 and swmm_time_for_clogging > 0.
- [ ] Export data (.DAT) files command: deletes previous SDCLOGGING.DAT if all fields swmm_clogging_factor and swmm_time_for_clogging values are 0 in Storm Drain Nodes table.
- [ ] Convert Schematic Layers to User Layers command: after converting schematic Storm Drain inlets into User Storm Drain Nodes, SDCLOGGING.DAT is read to complete fields swmm_clogging_factor and swmm_time_for_clogging.
- [ ] Import selected components files command (One Single Component): reads SDCLOGGING.DAT and updates Storm Drain Nodes tables/layer's fields swmm_clogging_factor and swmm_time_for_clogging.

**Comment by FLO-2DJJ** [comment]
@FLO-2DKaren, @FLO-2DNoemi
This issue is complete, but I’m waiting for pull request #1268 to be accepted before creating a pull request for this branch. The source of this branch depends on #1268.

**Comment by FLO-2DJJ** [comment]
Completed and merged into master from pull request #1270
